ui/src/components/footer/Footer.js

```diff
@@ -0,0 +1,316 @@
+import {
+  Link,
+  Container,
+  Image,
+  Text,
+  VStack,
+  Center,
+  Button,
+  Stack,
+  Flex,
+  Box,
+  Spacer,
+  HStack,
+  chakra,
+} from '@chakra-ui/react';
+import './SocialIcon.css';
+import { EmailIcon } from '@chakra-ui/icons';
+import { SocialIcon } from 'react-social-icons';
+
+const Footer = () => {
+  return (
+    <Container maxW="container.xl" py="2em">
+      <Flex w="100%" direction={['column', 'row']} py="2em">
+        <VStack w={['100%', '40%']} alignItems={['center', 'flex-start']}>
```

Comment:
space between items should appear wider according to design. similar situation for mobile

Current:
<img width="298" alt="image" src="https://github.com/user-attachments/assets/795c7ae7-c2b8-479c-ac4c-db7dc29c230b">

Design:
<img width="344" alt="image" src="https://github.com/user-attachments/assets/b721a39e-e363-4678-a662-e5b4111f5f47">


you can just add a `spacing={}` here to do this.

but to keep the logo and text close you will need to wrap it
```
<Stack flexDir="column">
  <Link href="/">
    <Image
      src={`${process.env.PUBLIC_URL}/images/ripple_blue.svg`}
      alt="Logo of HMCC"
      h="2em"
    />
  </Link>
  <Text
    color="#4A6EEB"
    fontSize="16"
    fontWeight="700"
    fontFamily="Manrope"
    wordWrap="break-word"
    textAlign={['center', 'left']}
  >
    Harvest Mission Community Church <br /> Hong Kong
  </Text>
</Stack>
```
something like that

Reply:
added padding to make it look more similar